charts/templates/settings.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "coral-credits.fullname" . }}
+  labels: {{ include "coral-credits.labels" . | nindent 4 }}
+type: Opaque
+# Use data because of https://github.com/helm/helm/issues/10010
+# Not doing so means that AWX-related keys are not removed on transition to the CRD
+data:
+  01-django.yaml: |
+    {{- tpl (.Files.Get "files/settings/01-django.yaml") . | b64enc | nindent 4 }}
```

Comment:
Personally, for the number of settings you have I would just inline them here.

Reply:
might just leave it as it is as we will probably add configuration in the future